README.md

```diff
@@ -40,7 +40,7 @@ class MyApp extends StatelessWidget {
         ),
         body: Center(
           child: TextField(
-            inputFormatters: [CurrencyTextInputFormatter()],
+            inputFormatters: [CurrencyTextInputFormatter.currency()],
```

Comment:
nit: this makes the code not backwards compatible and requires all users of the library to make same change for adopting the newer version, so maybe was better to keep the default constructor and make it a major version bump.